docs/UserGuide.md

```diff
@@ -211,7 +211,7 @@ and `johndoe@example.com` respectively.
 
 #### Finding a contact : `find`
 
-Finds persons whose names contain any of the given keywords. Also supports search with additional specifiers such as 
+Finds persons whose names contain any of the given keywords. This command also supports search with additional specifiers such as 
```

Comment:
'This command' is really not necessary. It is implied at the start of both sentences

Reply:
Not risking a debatable grammar error for no reason.